typescript/cli/src/config/ism.ts

```diff
@@ -196,7 +191,7 @@ export async function createRoutingConfig(
   }
   return {
     type,
-    owner: ownerAddress,
+    owner: '',
```

Comment:
Can you confirm, routing configs never have an owner now?

Reply:
only fallback has no owner:
https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/solidity/contracts/isms/routing/DefaultFallbackRoutingIsm.sol#L14-L30

new update/rebase reflects this